packages/date-picker/test/keyboard-input.test.js

```diff
@@ -87,49 +87,6 @@ describe('keyboard', () => {
     });
   });
 
-  describe('invalid date', () => {
-    it('should not select date on Enter if input invalid', async () => {
-      await open(datepicker);
-      await sendKeys({ type: 'foo' });
-      await sendKeys({ press: 'Enter' });
-      expect(datepicker.invalid).to.be.true;
-      expect(datepicker.checkValidity()).to.be.false;
-      expect(datepicker.value).to.equal('');
-      expect(input.value).to.equal('foo');
-    });
-
-    it('should validate on blur when not opened', async () => {
```

Comment:
This test is too complicated and is covered by other tests in this file that use `Tab`, and also this test:

https://github.com/vaadin/web-components/blob/2b0859f55b105278acf722dc25f8f5a5a52bbcc8/packages/date-picker/test/validation.test.js#L66-L68